.github/no-response.yml

```diff
@@ -12,5 +12,5 @@ closeComment: >-
   resolve this issue. We are therefore reluctantly going to close this
   bug for now. Please don't hesitate to comment on the bug if you have
   any more information for us; we will reopen it right away!
-  
+
```

Comment:
I suspect GitHub needs this - and that you may not have meant to include this change.

Reply:
Removed this change. I originally added this because I was failing a test that was checking for trailing whitespace